recipes/heatshrink/all/CMakeLists.txt

```
@@ -0,0 +1,31 @@
+cmake_minimum_required(VERSION 3.4)
+project(heatshrink C)
```

Don't forget to do `conan_basic_setup`.
```suggestion
project(heatshrink C)

include("${CMAKE_BINARY_DIR}/conanbuildinfo.cmake")
conan_basic_setup()
```